packages/components/src/FilterBar/context/types.ts

```diff
@@ -0,0 +1,13 @@
+export type FiltersValues = Record<string, any>
+
+export type FilterState<Id, Value> = {
```

Comment:
Are we trying to avoid using the word `FilterBar` in all of this for any particular reason?

Reply:
This is the State of a single Filter within the FilterBar. It would be `FilterBarFilterState` if you want to include it lol.